packages/twenty-front/src/modules/workflow/types/InputSchema.ts

```diff
@@ -0,0 +1,18 @@
+type InputSchemaPropertyType =
+  | 'string'
+  | 'number'
+  | 'boolean'
+  | 'object'
+  | 'array'
+  | 'unknown';
+
+type InputSchemaProperty = {
+  type: InputSchemaPropertyType;
+  enum?: string[] | undefined;
+  items?: InputSchemaProperty;
```

Comment:
without your explanation I would not have understand the diff between items and properties. I would prefer a more explicit name or at least a comment